packages/plugins/rubygem-foreman_acd/rubygem-foreman_acd.spec

```diff
@@ -1,35 +1,28 @@
 # template: foreman_plugin
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
-
 %global gem_name foreman_acd
 %global plugin_name acd
 %global foreman_min_version 2.1
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
+Name: rubygem-%{gem_name}
 Version: 0.9.4
-Release: 1%{?foremandist}%{?dist}
+Release: 2%{?foremandist}%{?dist}
 Summary: Foreman plugin to provide application centric deployment and self service portal
-Group: Applications/Systems
 License: GPLv3
 URL: https://www.orcharhino.com
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: git
-BuildRequires: git
-
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
-Requires: ruby
+Requires: ruby >= 2.5
 BuildRequires: foreman-assets >= %{foreman_min_version}
 BuildRequires: foreman-plugin >= %{foreman_min_version}
-BuildRequires: ruby
+Requires: ruby >= 2.5
+BuildRequires: ruby >= 2.5
 BuildRequires: rubygems-devel
 BuildArch: noarch
 Provides: foreman-plugin-%{plugin_name} = %{version}
-BuildRequires: rubygem(foreman_remote_execution) >= 7.0
-BuildRequires: rubygem(foreman-tasks) >= 7.0
+BuildRequires: rubygem(foreman_remote_execution) >= 3.3.0
+BuildRequires: rubygem(foreman-tasks) >= 0.10
```

Comment:
I guess some bumping inside the gemspec is in order? :)

Reply:
:see_no_evil:

Reply:
Funny thing is that is says

>   s.add_dependency 'foreman_remote_execution', '>= 8.0'
     s.add_dependency 'foreman-tasks', '>= 7.0'

in https://github.com/ATIX-AG/foreman_acd/blob/master/foreman_acd.gemspec

Reply:
But not in https://github.com/ATIX-AG/foreman_acd/blob/v0.9.4/foreman_acd.gemspec, which is what we package here.